### ISSUE TITLE
Auto-update functionality

### ISSUE THREAD

**Issue by dbkaplun** [zz85/space-radar#13]

**Comment by zz85** [comment]
Are you referring to the visualizations auto updating, or are you referring to have the app auto update itself when there are updates?

**Comment by dbkaplun** [comment]
I meant having the app auto-update itself when there are updates. Is that feasible?

**Comment by zz85** [comment]
Yes, that is possible but I'll need to look into how I can do it. From what it seems, it might require bundling it into the installer.

**Comment by dbkaplun** [comment]
I found this: https://www.npmjs.com/package/electron-updater

**Comment by zz85** [comment]
Thanks for the link, I'll give it a try!

**Comment by zz85** [comment]
For now I'm quite happy with automatic releases with electron-packager (with the exception of tweaking Mac builds), but I would probably want to look into https://github.com/electron-userland/electron-builder the next round..

**Comment by zz85** [comment]
Electron now have an official service for self updating apps https://electronjs.org/blog/autoupdating-electron-apps

**Comment by zz85** [comment]
As part of [v6](https://github.com/zz85/space-radar/releases/tag/v6.0.0), I've integrated updated with electron-updater which should work in theory the next update, but I'll leavet his open for now
